test/functional/wallet_assumeutxo.py

```diff
@@ -36,18 +32,19 @@ def add_options(self, parser):
 
     def set_test_params(self):
         """Use the pregenerated, deterministic chain up to height 199."""
-        self.num_nodes = 3
+        self.num_nodes = 4
         self.rpc_timeout = 120
         self.extra_args = [
             [],
             [],
             [],
+            ["-fastprune", "-prune=1"]
```

Comment:
nit, for smaller future diff if another node is added
```suggestion
            ["-fastprune", "-prune=1"],
```